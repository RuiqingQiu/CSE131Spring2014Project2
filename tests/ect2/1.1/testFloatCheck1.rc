//This file is trying to test float literals, constants, varibles
float test1;
const float test2 =  10.0;
static const float test3 = test2;

function  : void main(){
    float test4 = +-23.9;
    float test5;
    cout << 1000.000 << endl;
    cout << -98.238 << endl;
    cout << test1 << endl;
    cout << test2 << endl;
    cout << test3 << endl;
    cout << test4 << endl;
    cout << test5 << endl;
}

