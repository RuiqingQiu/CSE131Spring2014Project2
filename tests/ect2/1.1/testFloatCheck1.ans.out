1000.00
-98.24
0.00
10.00
10.00
-23.90
0.00
